Solutions/LargestPrimeFactor.cs

```diff
@@ -0,0 +1,36 @@
+namespace Solutions;
+
+public class LargestPrimeFactor : ProblemSolverBase
+{
+  public LargestPrimeFactor()
+  {
+    ProblemID = 3;
```

Comment:
I like the idea, can you also include the problem description from project euler website directly here?